docs/lib-documentation.md

```diff
@@ -304,35 +269,28 @@ _Found a bug? File it on [the repo](https://github.com/kurtosis-tech/kurtosis-cl
 <!-- TODO Make the function definition not include args or return values, so we don't get these huge ugly links that break if we change the function signature -->
 <!-- TODO make the reference names a) be properly-cased (e.g. "Service.isAvailable" rather than "service_isavailable") and b) have an underscore in front of them, so they're easy to find-replace without accidentally over-replacing -->
 
-[containercreationconfig]: #containercreationconfig
-[containercreationconfig_usedports]: #setstring-usedports
-[containercreationconfig_filegeneratingfuncs]: #mapstring-funcfile-filegeneratingfuncs
-[containercreationconfig_filesartifactmountpoints]: #mapstring-string-filesartifactmountpoints
-[containercreationconfig_usedstaticfiles]: #setstaticfileid-usedstaticfiles
+[containerconfig]: #containerconfig
```

Comment:
Love these updates. Just make sure to test that all these paths work, because unfortunately our Markdown link-checker can't catch these `#thing` anchor links.

To test, I've installed the https://chrome.google.com/webstore/detail/markdown-viewer/ckkdlimhmcjmikdlpkmbgfkaikojcbjk Chrome extension and click the links.

This is also how I generate the anchor links in the first place - click a header, then copy from the URL

Reply:
yep! I did it that way and I've checked them all.